### ISSUE TITLE
Alt text is missing in grid card images

### ISSUE THREAD

**Issue by Valerie-ts** [Neoteroi/mkdocs-plugins#63]
Hi,
I hope someone reads this. I realized that the images used in the Neoteroi grid cards do not have any default alt text nor is there an option to provide alt text. This negatively impacts the SEO of a website.
Icons do have an alt text. I was wondering if this could be added in the future?

Sunny greetings :)

**Comment by RobertoPrevato** [comment]
Hi! Thank You for reporting this issue. The good news is that specifying the `alt` attribute for images is already supported. The documentation is lacking here, it must be improved.

If you specify an object for the `image` property instead of a string, like here:

```
::cards::

- title: Zeus
  content: Lorem ipsum dolor sit amet.
  image:
    url: ./img/icons/001-zeus.png
    alt: A picture of Zeus

- title: Athena
  content: Lorem ipsum dolor sit amet.
  image:
    url: ./img/icons/003-athena.png
    alt: A picture of Athena

::/cards::
```

You obtain HTML with proper `img` elements including `alt` property.

By the way, I realized the library can be improved to automatically populate the `alt` property with the card title, when it's not specified.
